### PR TITLE
Workaround reorganization

### PR DESCRIPTION
**Pull request by antoine-levitt**

### CODE REVIEW

File: src/workarounds/README.md [hunk 1]
```diff
@@ -0,0 +1 @@
+Workarounds, fixes and hacks for various external libraries, should ideally be upstreamed.
```
Comment (mfherbst) [inline src/workarounds/README.md:1]: add no warranty statement

**Comment by mfherbst** [comment]
Could you grep for FourierTransforms.jl in the full project. I made all sorts of exceptions for this folder (e.g. coverage). Just change these as well

**Comment by antoine-levitt** [comment]
I don't see anything?

**Comment by mfherbst** [comment]
Ah yes, I changed that at some point, I remember now. Sorry ... but good that you checked again

**Pull request merged**